aptos-node/src/lib.rs

```diff
@@ -558,7 +559,7 @@ pub fn setup_environment_and_start_node(
     let admin_service = services::start_admin_service(&node_config);
 
     // Set up the storage database and any RocksDB checkpoints
-    let (aptos_db, db_rw, backup_service, genesis_waypoint) =
+    let (_aptos_db, db_rw, backup_service, genesis_waypoint) =
```

Comment:
Why don't we need _aptos_db anymore?

Reply:
So the existing way was redundant?

Reply:
nit: if this is the case, maybe just remove the DbReader return value from `initialize_database_and_checkpoints` altogether?

Also @areshand to confirm? 😄